handlers/main.yml

```diff
@@ -18,3 +18,8 @@
   service:
     name: ufw
     state: restarted
+
+- name: save iptables rules (Debian/Ubuntu and CentOS/RHEL/Fedora)
+  command: "{{ iptables_save_command }}"
+  when: ansible_os_family == 'Debian' or ansible_os_family == 'RedHat'
```

Comment:
Since we're already changing the command based on the OS, do we still need the `when` clause? It also looks like it won't let Ubuntu save iptables either, since it's not included.

Reply:
@kyl191 Ubuntu is Debian-based so is passes as the ansible_os_family 'Debian'. So this line is going to run for Debian, Ubuntu, CentOS, RHEL and Fedora. That's almost every OS but I left the 'when' here for unknown situations like Solaris, etc.